src/Confluent.Kafka/Impl/LibRdKafka.cs

```diff
@@ -709,24 +731,22 @@ private class NativeMethods
             internal static extern ErrorCode rd_kafka_position(
                     IntPtr rk, IntPtr partitions);
 
+            // note: producev signature is rd_kafka_producev(rk, ...)
+            // __arglist is an undocumented keyword which deal with it
+            // but doesn't work on standardlib (and undocumented so don't use)
+            // stay simple with one binding for now, but it will be worth benchmarking
+            // the overload with no timestamp, opaque, partition, etc
             [DllImport(DllName, CallingConvention = CallingConvention.Cdecl)]
-            internal static extern IntPtr rd_kafka_produce(
-                    IntPtr rkt,
```

Comment:
There already is a rkt overload in producev. I didn't use it cause I thought we wanted get rid of the rkt (from comments on getTopicHandle) but we will need it anyway if we want to pass a custom topic on a topic. Will update the PR